local/start-flow.sh

```diff
@@ -40,6 +39,19 @@ function bail() {
 # verify requirements here, before we get into the tmux session.
 command -v nc || bail "netcat must be installed (nc command was not found)"
 
+function copy_local_flowctl_config() {
```

Comment:
Just moving this function back to the way it used to be - in `start-flow.sh` and not called with any particular argument.